runtime/src/main/java/org/corfudb/runtime/view/LayoutManagementView.java

```diff
@@ -415,8 +416,13 @@ public void reconfigureSequencerServers(Layout originalLayout, Layout newLayout,
                         || !originalLayout.getPrimarySequencer()
                         .equals(newLayout.getPrimarySequencer())) {
 
-                    StreamsAddressResponse streamsAddressesResponse = runtime.getAddressSpaceView()
-                            .getLogAddressSpace();
+                    // The sequencer state needs to be retrieved from the head logunit from the
+                    // new layout. Note that AddressSpaceView::getLogAddressSpace shouldn't be
```

Comment:
Can we remove AddressSpaceView::getLogAddressSpace? That method was only used here and in a couple of unit tests, since we are now using Utils instead, we can remove that method and also change the unit tests to use Utils as well.

Reply:
Done.